runelite-client/src/main/java/net/runelite/client/plugins/blowpipe/BlowpipeInfoOverlay.java

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Is this plugin needed? There's supply tracker (and if I recall correctly also ammo plugin). That and making sure your blowpipe has enough shit in it is a once per day thing at most.